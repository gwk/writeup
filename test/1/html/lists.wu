writeup v0

# Intro

t1.
* a
  a1
  * b
    b1
  a2
t2.
t3.
