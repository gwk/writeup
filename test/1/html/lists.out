<section class="S1" id="s0">
  <h1 id="h0">Intro</h1>
  <p>
    t1.
  </p>
  <ul class="L1">
    <li>
      <p>
        a
      <br />
        a1
      </p>
      <ul class="L2">
        <li>
          b
        <br />
          b1
        </li>
      </ul>
      <p>
        a2
      </p>
    </li>
  </ul>
  <p>
    t2.
  <br />
    t3.
  </p>
</section>
